website/source/docs/internals/architecture.html.md

```diff
@@ -12,61 +12,12 @@ Consul is a complex system that has many different moving parts. To help
 users and developers of Consul form a mental model of how it works, this
 page documents the system architecture.
 
-~> **Advanced Topic!** This page covers technical details of
-the internals of Consul. You don't need to know these details to effectively
-operate and use Consul. These details are documented here for those who wish
-to learn about them without having to go spelunking through the source code.
-
 ## Glossary
```

Comment:
The Glossary heading here looks a little goofy since the section is so short now. Combine the glossary recommendation with the first section. Maybe highlight as a `->`-style note?

Reply:
Yes, that would probably look a lot better!